### PR TITLE
E1673. Refactor [question_type].rb (eg. criterion.rb)

### PR DESCRIPTION
**Pull request by zhoilet**

### CODE REVIEW

**Comment by yangsong8** [comment]
Plz confirm your project number: is it 1673?

**Comment by zhoilet** [comment]
Yes, the complete title is "E1673. Refactor [question_type].rb" @yangsong8

**Comment by yangsong8** [comment]
Okay, will start test this project shortly.

**Comment by zhoilet** [comment]
Thank you! We are looking forward to your feedback! @yangsong8

File: app/models/scale.rb [hunk 1]
```diff
@@ -1,75 +1,167 @@
...
+    html = edit_remove_button(count) + edit_seq(count)
+    html += edit_question_type(count) + edit_weight(count)
+    html += edit_max_label(count) + edit_min_label(count)
+    safe_join([raw("<tr>"), raw("</tr>")], raw(html))
+  end
+
+  def edit_remove_button(_count)
+    html = '<td align="center"><a rel="nofollow" data-method="delete" href="/questions/'
+    html += self.id.to_s + '">Remove</a></td>'
+    html += '<td><input size="6" value="' + self.seq.to_s + '" name="question['
```
Comment (Winbobob) [inline app/models/scale.rb]: Weird split!

File: app/models/scale.rb [hunk 2]
```diff
@@ -1,75 +1,167 @@
...
+
+    html
+  end
+
+  def edit_seq(_count)
+    html = self.id.to_s + '][seq]" id="question_' + self.id.to_s + '_seq" type="text"></td>'
+    html += '<td><textarea cols="50" rows="1" name="question[' + self.id.to_s + '][txt]" id="question_'
```
Comment (Winbobob) [inline app/models/scale.rb]: Weird split!

**Comment by Winbobob** [comment]
Merge 2 files `checkbox.rb` and `criterion.rb` only.

**Pull request merged**